Makefile.include

```diff
@@ -367,6 +367,15 @@ include $(RIOTMAKE)/defaultmodules.inc.mk
 # Include Kconfig functionalities
 include $(RIOTMAKE)/kconfig.mk
 
+# Include Board and CPU configuration
+INCLUDES += $(addprefix -I,$(wildcard $(BOARDDIR)/include))
+include $(BOARDDIR)/Makefile.include
+INCLUDES += -I$(RIOTCPU)/$(CPU)/include
+include $(RIOTCPU)/$(CPU)/Makefile.include
+
+# Import all toolchain settings
+include $(RIOTMAKE)/toolchain/$(TOOLCHAIN).inc.mk
```

Comment:
This is reverting a long effort of changing the order of inclusion: https://github.com/RIOT-OS/RIOT/issues/9913.https://github.com/RIOT-OS/RIOT/issues/9913

Reply:
we can also have a error message telling the user that he hast to blacklist the provided but not exiting toolchain that the dependency solver prefered for a reason unknown to the user (maybe it was listed first somewhere, or it is alphanumerical first)